docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -148,6 +148,38 @@ An encrypted `filesystem-realm` that makes use of the `credential-store` could b
 /subsystem=elytron/filesystem-realm=exampleFsRealm:add(path=fs-realm-users,relative-to=jboss.server.config.dir, credential-store=mycredstore, secret-key=key)
 ----
 
+==== Integrity
+
+It is also possible to enable integrity support on the filesystem realm by configuring the key-store and key-store-alias attributes for the realm.
+
+When enabling integrity on a filesystem realm, you will have to specify the following 2 attributes
+
+* `key-store`: This attribute specifies the key-store where the key-pair (Private Key and Public Key) is used for signing each identity file for the filesystem realm.
+
+* `key-store-alias`: This attribute specifies the alias of the key-store that contains the correct key-pair to be using for signing the files.
```

Comment:
the alias within the key-store that identifies the PrivateKeyEntry to use to sign identity files and perform filesystem integrity checks